packages/js-sdk/src/sandbox/index.ts

```diff
@@ -317,6 +317,22 @@ export class Sandbox extends SandboxApi {
     await Sandbox.kill(this.sandboxId, { ...this.connectionConfig, ...opts })
   }
 
+  /**
+   * Get the timeout of the sandbox.
+   *
+   * @param opts connection options.
+   *
+   * @returns timeout of the sandbox.
```

Comment:
This returns the "end time", no?? 
i understand you want the naming to be consistent with `setTimeout` but to me "timeout" means the "delay".